tests/execution/grpc-batch.md

```diff
@@ -81,14 +81,14 @@ type News {
 ```yml
 - request:
     method: POST
-    url: http://localhost:50051/news.NewsService/GetMultipleNews
+    url: http://localhost:50051/NewsService/GetMultipleNews
     body: \0\0\0\0\n\x02\x08\x02\n\x02\x08\x03
   response:
     status: 200
     body: \0\0\0\0t\n#\x08\x02\x12\x06Note 2\x1a\tContent 2\"\x0cPost image 2\n#\x08\x03\x12\x06Note 3\x1a\tContent 3\"\x0cPost image 3
 - request:
     method: POST
-    url: http://localhost:50051/news.NewsService/GetMultipleNews
+    url: http://localhost:50051/NewsService/GetMultipleNews
```

Comment:
Should not change existing tests.

Reply:
this test was wrongly changed before. the way it should work is /<service>/<method>

Reply:
I catch this when looking into the @call operator tests. It was working and needed this change after the merge (because a bug was introduced earlier on)

Reply:
package name could be the part of url actually if it's defined in the proto and it's defined in proto file.

take a look into

Reply:
do you have any example of it that I can add to the tests?